### ISSUE TITLE
Non-existent Geo-Objects show up in lists with only the code and all other attributes blank

### ISSUE THREAD

**Issue by rfromthecastle** [terraframe/geoprism-registry#781]
**List the CGR information**
- CGR version: 0.19.2
- CGR instance (e.g. a URL): https://staging-georegistry.geoprism.net/cgr

**Describe the bug (clear and concise)**
After making a Geo-Object non-existent from a certain date, when viewing a published list with a validity date after the Geo-Object has been marked as non-existent, the Geo-Object is still included in the list, but with only the code listed and everything else blank.

**To Reproduce**
1. As a Registry Maintainer, mark a Geo-Object as non-existent from a certain date.
2. Publish a list for a validity date/period after the Geo-Object is non-existent.
3. View the list.

**Expected behavior (clear and concise)**
The Geo-Object does not show up in the list for validity periods when the Geo-Object is non-existent.

**Screenshots**
![image.png](https://images.zenhubusercontent.com/5e5cdb3c03758d213c13e15f/2ea17e99-5bef-4a94-9223-0065dae09a3c)
See Geo-Object H00101

**Desktop (please complete the following information):**
 - OS: Windows 11
 - Browser: Google Chrome
 - Version: 101.0.4951.64

**Additional context (if any)**
N/A

**Comment by SteeveEbener** [comment]
Duplicate of #790. I let you decide which one to keep based on the description of each ticket. The problem does not onlyappear only after setting a geo-object as non-existent

**Comment by rfromthecastle** [comment]
Working in v0.19.5 on Staging

**Comment by SteeveEbener** [comment]
I would like to suggest that we close this ticket and keep #790 open
